stages/org.osbuild.rpm

```diff
@@ -303,13 +303,17 @@ def main(tree, inputs, options):
 
     for filename, data in packages.items():
         if data.get("rpm.check_gpg"):
-            subprocess.run([
-                "rpmkeys",
-                *rpm_args,
-                "--root", tree,
-                "--checksig",
-                filename
-            ], cwd=pkgpath, stdout=subprocess.DEVNULL, check=True)
+            try:
+                subprocess.run([
+                    "rpmkeys",
+                    *rpm_args,
+                    "--root", tree,
+                    "--checksig",
+                    filename
+                ], cwd=pkgpath, stdout=subprocess.DEVNULL, check=True)
+            except Exception:
```

Comment:
I believe it's better to catch `subprocess.CalledProcessError` directly.

Reply:
+1

Reply:
Actually, that was on purpose, since it isn't just catching the error but also re-raising it. The goal is to add context to whatever error happens to come along (admittedly, usually a CalledProcessError). If it only catches on type any other type will be missing the hint as to what happened.

Reply:
Maybe. Probably. But I don't want to depend on that. The point is to give the user enough information about what's happening, these failures have been a big pain to track down.